README.md

```diff
@@ -1,4 +1,5 @@
-![Tasmota logo](/tools/logo/TASMOTA_FullLogo_Vector.svg)
+This is a FORK providing ( in a few days I hope ) the translation file for the Catalan language.
```

Comment:
This should not be changed. Please, revert it. Thanks.

Reply:
Reverted. Apologize for the inconvenience.